src/main/java/uk/gov/hmcts/reform/hois/domain/entities/HomeOfficeResponse.java

```diff
@@ -0,0 +1,77 @@
+package uk.gov.hmcts.reform.hois.domain.entities;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import java.util.Arrays;
+import lombok.Data;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@Data
+public class HomeOfficeResponse {
+    /* @JsonProperty("ho_reference")
```

Comment:
you can safely clean up these commented lines. You can see the history and bring it up at any point if needed.

Reply:
Removed as suggested